resources/views/layouts/layout.blade.php

```diff
@@ -55,6 +59,145 @@
+                        Hoofdpagina
+                    </a></li>
+                    <li><a class="dropdown-item text-center" href="/training">
+                        Trainingen
+                    </a></li>
+                    <li><a class="dropdown-item text-center" href="/evenement">
+                        Evenementen
+                    </a></li>
+                    <li><a class="dropdown-item text-center" href="/galerij/2023">
+                        2023
+                    </a></li>
+                    <li><a class="dropdown-item text-center" href="/galerij/2022">
+                        2022
+                    </a></li>
+                    <li> <a class="dropdown-item text-center" href="/galerij/2021">
+                        2021
+                    </a></li>
+                    <li><a class="dropdown-item text-center" href="/faq">
+                        FAQ
+                    </a></li>
+                    <li><a class="dropdown-item text-center" href="/nieuwsbrief">
+                        Nieuwsbrief
+                    </a></li>
+                    <li><a class="dropdown-item text-center" href="/team">
+                        Team
+                    </a></li>
+                    <li><a class="dropdown-item text-center" href="/partner">
+                        Partner
+                    </a></li>
+                    <li><a class="dropdown-item text-center" href="/overons">
+                        Over Ons
+                    </a></li>
+                    <li><a class="dropdown-item text-center" href="/locatie">
+                        Locatie
+                    </a></li>
+                    <li><a class="dropdown-item text-center" href="/links">
+                        Links
+                    </a></li>
+                </ul>
+            </form>
+        </div>
+        </nav>
+
+        <div class="container">
+            @yield('content')
+        </div>
+        <footer>
+          <div class="footer-content bg-light">
+           <div class="footer-info">
+          <p class="text-dark">Secretariaat Stichting Special Golf: specialgolfhaverleij@gmail.com</p>
+          <p class="text-dark">Rekeningnummer Rabobank: NL 38 RABO 0118102206 o.v.v. Special Golf</p>
+          <p class="text-dark">KvK nr. 88714543 - RSIN 864744328 - ANBI</p>
+          </div>
+          <ul class='socials text-dark'>
+             <li><a class="text-dark" aria-label="Icoontje met link naar het Facebook account van Special Golf Haverlij" href=https://www.facebook.com/DeHaverleij/?locale=nl_NL  ><i class="fa-brands fa-facebook" ></i></a></li>
+             <li><a class="text-dark" aria-label="Icoontje met link naar het Instagram account van Special Golf Haverlij" href=https://www.instagram.com/specialgolf2021/  ><i class="fa-brands fa-instagram"></i></a></li>
+          </ul>
+
+          </div>
+        </footer>
+    </body>
+</html>
+
+@else
+
```

Comment:
Beetje dubbel. Maar dit oplossen hoort vgm ook meer bij user story 137

Reply:
fixed, de navbar staat nu alleen in de if/else